src/segments/project.go

```diff
@@ -163,3 +176,29 @@ func (n *Project) getNuSpecPackage(item ProjectItem) (string, string) {
 
 	return data.MetaData.Version, data.MetaData.Title
 }
+
+func (n *Project) getDotnetProject(item ProjectItem) (string, string) {
+	files := n.env.LsDir(n.env.Pwd())
+	var name string
+	var content string
+	// get the first match only
+	for _, file := range files {
+		extension := filepath.Ext(file.Name())
+		if extension == ".csproj" || extension == ".fsproj" || extension == ".vbproj" {
+			name = strings.TrimSuffix(file.Name(), filepath.Ext(file.Name()))
+			content = n.env.FileContent(file.Name())
+			break
+		}
+	}
+	// the name of the parameter may differ depending on the version,
+	// so instead of xml.Unmarshal() we use regex:
+	tag := "(?P<tag><.*TargetFramework.*>(?P<tfm>.*)</.*TargetFramework.*>)"
```

Comment:
Can we make `tag` and `tfm` uppercase? Let's not forget to also change below in that case.
```suggestion
	tag := "(?P<TAG><.*TargetFramework.*>(?P<TFM>.*)</.*TargetFramework.*>)"
```